includes/Registry.php

```diff
@@ -35,7 +35,7 @@ public function get_relationship_key( $from, $to, $name ) {
 		sort( $to );
 		$to = implode( '.', $to );
 
-		return "{$from}_{$to}_{$name}";
+		return "{$from}~{$to}~{$name}";
```

Comment:
Switched to a more obscure delimiter to avoid confusion with underscores in post type names.